src/r5py/util/check_od_data_set.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
```

Comment:
Might we also have other "checks" for files/inputs etc? Just thinking that should we have a more generic filename into which we could possibly add also other checks in the future. The file could e.g. be `validate_input.py` or something along those lines..

Reply:
that’s true :+1:

Reply:
I went even bigger and called the file `data_validation.py`